env_docker.go

```diff
@@ -129,10 +130,10 @@ func New(opts ...EnvironmentOption) (_ *DockerEnvironment, err error) {
 		return nil, errors.Wrapf(err, "create docker network '%s'", d.networkName)
 	}
 
-	switch runtime.GOOS {
-	case "darwin":
-		d.hostAddr = dockerMacOSGatewayAddr
-	default:
+	switch e2einteractive.HostOSPlatform() {
```

Comment:
Let's make sure main package is not importing side code. e2e interactive should be imported only if you want interactivity. Let's have rather e2einteractive use e2e package or create a separate package.

Reply:
Moved to a separate package.